src/kernel/src/common/startup.s

```diff
@@ -6,14 +6,11 @@ _Reset:
     // Disable other cores
     mrc p15, #0, r1, c0, c0, #5
     and r1, r1, #3
-    cmp r1, #0
+    cmp r1, #3
```

Comment:
what's the advantage of 3 here?

Reply:
Though I have not been able to obtain a multi-core Raspberry Pi to verify, everything indicates that newer versions of the RPi bootloader disable all cores except one upon entry. However, in older bootloader versions and in QEMU this is not the case. Older versions of the linux kernel which had to deal with multiple cores being active act as you see here.

The co-processor register that is being read is not documented on the ARM1176. However, it happens to return this value.

Without this change, as you can imagine an Rpi Zero would be stuck forever.